casper/src/main/scala/coop/rchain/casper/rholang/RuntimeSyntax.scala

```diff
@@ -569,9 +570,9 @@ final class RuntimeOps[F[_]: Sync: Span: Log](
       )(validatorsPar => validatorsPar.size == 1)
       .map(validatorsPar => toValidatorSeq(validatorsPar.head))
 
-  def computeBonds(hash: StateHash): F[Seq[Bond]] =
+  def computeBonds(hash: StateHash, shardId: String): F[Seq[Bond]] =
     // Create a deploy with newly created private key
-    playExploratoryDeploy(bondsQuerySource, hash)
+    playExploratoryDeploy(bondsQuerySource, hash, shardId)
```

Comment:
If this is the only reason to add parameter to RuntimeManager than we don't need it.

Reply:
Exactly. I removed parameter in [commit](https://github.com/rchain/rchain/pull/3631/commits/6b3323cd4d0558d4385132b75632a09a1bcaad60).